npm/translatePolicy.go

```diff
@@ -189,6 +189,7 @@ func translateIngress(ns string, policyName string, targetSelector metav1.LabelS
 		fromRuleExists := false
 		addedPortEntry = addedPortEntry || portRuleExists
 		ipCidrs[i] = make([]string, len(rule.From))
+		comment := ns + "/" + policyName
```

Comment:
not aware of a delimiter after policyName, we should potentially add some separation for legibility

comment might read as "ns/policynameALLOW-ALL-TO-targetselectorComment"